pandas/tests/indexes/test_multi.py

```diff
@@ -2297,11 +2327,50 @@ def test_level_setting_resets_attributes(self):
         ind = MultiIndex.from_arrays([
             ['A', 'A', 'B', 'B', 'B'], [1, 2, 1, 2, 3]
         ])
-        assert ind.is_monotonic
+        self.assertTrue(ind.is_monotonic)
         ind.set_levels([['A', 'B', 'A', 'A', 'B'], [2, 1, 3, -2, 5]],
                        inplace=True)
+
         # if this fails, probably didn't reset the cache correctly.
-        assert not ind.is_monotonic
+        self.assertFalse(ind.is_monotonic)
+
+    def test_is_monotonic(self):
```

Comment:
also include some tests for `is_monotonic_increasing` and `is_monotonic_decreasing` ?